csp-mixin/signals/gcpceOverview.libsonnet

```
@@ -0,0 +1,300 @@
+local commonlib = import 'common-lib/common/main.libsonnet';
+function(this)
+  {
+    local s = self,
+    filteringSelector: 'job!=""',
+    groupLabels: ['job', 'project_id'],
+    instanceLabels: [],
+    aggLevel: 'instance',
+    discoveryMetric: {
+      stackdriver: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization',
+    },
+    signals: {
+      instanceCount: {
+        name: 'Instance count',
+        description: 'Number of VM instances',
+        type: 'raw',
+        unit: 'short',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name) (stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s})',
+            legendCustomTemplate: '',
+            aggKeepLabels: ['instance_name'],
+            exprWrappers: [['count(', ')']],
+          },
+        },
+      },
```

You can use **type=gauge** and remove extras from **expr** and you will have the same result because an instance cannot be in 2 different job and project at the same time. I tested and works

```suggestion
     instanceCount: {
        name: 'Instance count',
        description: 'Number of VM instances',
        type: 'gauge',
        unit: 'short',
        sources: {
          stackdriver: {
            expr: 'stackdriver_gce_instance_compute_googleapis_com_instance_cpu_utilization{%(queriesSelector)s}',
            legendCustomTemplate: '',
            aggKeepLabels: ['instance_name'],
            exprWrappers: [['count(', ')']],
          },
        },
      },
```